plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/struct/DBSVisibilityScopeProvider.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jkiss.dbeaver.model.struct;
+
+import org.jkiss.code.NotNull;
+import org.jkiss.dbeaver.DBException;
+import org.jkiss.dbeaver.model.runtime.DBRProgressMonitor;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Provides additional globally visible scopes
+ */
+public interface DBSVisibilityScopeProvider {
+    /**
+     * Returns additional globally visible scopes, like information_schema
+     */
+    default List<DBSObjectContainer> getPublicScopes(@NotNull DBRProgressMonitor monitor) throws DBException {
+        return Collections.emptyList();
+    }
```

Comment:
Does it really need a default implementation? It seems all its current implementors have it implemented.